packages/server/src/api/controllers/query/import/sources/curl.ts

```diff
@@ -2,9 +2,8 @@ import { ImportSource, ImportInfo } from "./base"
 import { Query } from "../../../../../definitions/common"
 import { URL } from "url"
 
-const curlconverter = require("curlconverter")
-
-const parseCurl = (data: string): any => {
+const parseCurl = async (data: string): Promise<any> => {
+  const curlconverter = await import("curlconverter")
```

Comment:
out of interest, why is it necessary to import this way? Is there a problem with a top level import like `import curlconverter from "curlconverter"`

Reply:
Yep, the later versions of the `curlconverter` package use top level `await`, and you can't just import it normally without changing `"type": "module"` in `package.json`. This is actually causing the rest tests to fail, which I'm looking at atm

Reply:
Very interesting - didn't realise this would happen with top level awaits, good to know that they make the library a lot less usable!